cms/templates/settings.html

```diff
@@ -364,10 +364,72 @@ <h2 class="title-2">${_("Introducing Your Course")}</h2>
                       <input type="text" dir="ltr" class="long new-course-image-url" id="course-image-url" value="" placeholder="${_("Your course image URL")}" autocomplete="off" />
                       <span class="tip tip-stacked">${_("Please provide a valid path and name to your course image (Note: only JPEG or PNG format supported)")}</span>
                     </div>
-                    <button type="button" class="action action-upload-image">${_("Upload Course Image")}</button>
+                    <button type="button" class="action action-upload-image" id="upload-course-image">${_("Upload Course Image")}</button>
                   </div>
                 </li>
 
+                % if enable_extended_course_details:
+                <li class="field image" id="field-hero-image">
+                  <label for="hero-image-url">${_("Course Details Background")}</label>
+                  <div class="current current-course-image">
+                    % if context_course.hero_image:
+                    <span class="wrapper-course-image">
+                      <img class="course-image" id="hero-image" src="${hero_image_url}" alt="${_('Hero Image')}"/>
+                    </span>
+
+                    <span class="msg msg-help">
+                    ${_("You can manage this image along with all of your other <a href='{}'>files &amp; uploads</a>").format(upload_asset_url)}
+                    </span>
+
+                    % else:
+                    <span class="wrapper-course-image">
+                      <img class="course-image placeholder" id="hero-image" src="${hero_image_url}" alt="${_('Hero Image')}"/>
+                    </span>
+                    <span class="msg msg-empty">${_("Your course currently does not have an image. Please upload one (JPEG or PNG format, and minimum suggested dimensions are 375px wide by 200px tall)")}</span>
```

Comment:
Are these actually the preferred dimensions for this background image? It's much wider.

Also note that there is no period at the end of the second sentence.